CLAUDE.md

```
@@ -1,7 +1,27 @@
-# CLAUDE.md
+# CLAUDE.md - CLDCDE Project Instructions
 
 This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
 
+## Prompt Hook
+
+When responding to any user prompt that is **more than 3 sentences**, append the following hook at the end:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)